tests/e2e/pageobjects/login/openshift/OcpUserLoginPage.ts

```diff
@@ -22,7 +22,7 @@ export class OcpUserLoginPage implements IOcpLoginPage {
 	async login(): Promise<void> {
 		Logger.debug();
 
-		if (OAUTH_CONSTANTS.TS_OCP_LOGIN_PAGE_PROVIDER_TITLE !== '') {
+		if (await this.ocpLogin.isIdentityProviderLinkVisible()) {
```

Comment:
If I understood it correctly, after this changes it will the test will ignore "OAUTH_CONSTANTS.TS_OCP_LOGIN_PAGE_PROVIDER_TITLE" value and therefore will not be able to log into the cluster with cluster admin credentials when identity provider link is visible.

Did you try to fix the following problem described in issue https://github.com/eclipse/che/issues/22823 by setting up `TS_OCP_LOGIN_PAGE_PROVIDER_TITLE = ''` when run the test?
> In the case when is used OCP cluster without OAuth config the test DevConsoleIntegration is failure.

Reply:
All E2E tests use the logic, which I try to do here for one test `DevConsoleIntegration`.
Yes the logic is not ideal..but when I asked in MR about this solution nobody had objections..

https://gitlab.cee.redhat.com/codeready-workspaces/crw-jenkins/-/merge_requests/1815#note_9848066
https://gitlab.cee.redhat.com/codeready-workspaces/crw-jenkins/-/merge_requests/1815#note_9848409

Now I should close this PR and return back to new crw-jenkins MR

Reply:
@artaleks9 : sorry, I forgot to check `htpasswd` identity provider clean up when review MR https://gitlab.cee.redhat.com/codeready-workspaces/crw-jenkins/-/merge_requests/1815

Reply:
> All E2E tests use the logic
then they are using incorrect logic which worth to fix, IMHO

Reply:
@dmytro-ndp 
So, need to list possible login options in order to clarify where and how implement it.
Login to Che Eclipse can be include:
* **minikube** (is actual ?) 
   -- user-name / user-password _login page_
* **OCP cluster without `Identity Provider`:**
   -- user-name / user-password _login page_ 
* **OCP cluster with `Identity Provider`**:
   -- identity-provider-name _login page_
   -- user-name / user-password _login page_

-----------

Actually the code of `DevConsoleIntegration` E2E test has login OCP cluster link  and uses another implemenation, but really, it the same if to compare with login to Che.

Reply:
@artaleks9 : I have reconsidered opinion and it seems to be acceptable decision in PR if it's needed to have `TS_OCP_LOGIN_PAGE_PROVIDER_TITLE='kube:admin'` parameter in case of identity provider has been added.